app/templates/components/build-header.hbs

```diff
@@ -45,11 +50,19 @@
         </li>
       {{/if}}
       <li>
-        {{#external-link-to href=urlGitHubBranch class="commit-branch-url"}}
-          {{svg-jar 'icon-branch' class="icon"}}
-          <span data-tooltip="View branch on GitHub" class="label-align inner-underline">Branch {{build.branchName}}</span>
-          {{svg-jar 'icon-external-link' class="icon"}}
-        {{/external-link-to}}
+        {{#if item.isTag}}
+          {{#external-link-to href=urlGitHubTag class="commit-branch-url"}}
+            {{svg-jar 'icon-tag' class="icon"}}
+            <span data-tooltip="View tag on GitHub" class="label-align inner-underline">Tag {{build.tag.name}}</span>
+            {{svg-jar 'icon-external-link' class="icon"}}
+          {{/external-link-to}}
+        {{else}}
+          {{#external-link-to href=urlGitHubBranch class="commit-branch-url"}}
+            {{svg-jar 'icon-branch' class="icon"}}
+            <span data-tooltip="View branch on GitHub" class="label-align inner-underline">Branch {{build.branchName}}</span>
+            {{svg-jar 'icon-external-link' class="icon"}}
+          {{/external-link-to}}
+        {{/if}}
```

Comment:
Seeing all these `if item.isTag`/`else`s spread about brings to mind a refactoring pattern, [Replace Conditional with Polymorphism](https://refactoring.guru/replace-conditional-with-polymorphism). I don’t know if it’s an exact match, but it could be a way to reduce some duplication. Maybe it’s not worth it at this point because there is only one conditional, not a series.

But as an example, this template could be:

```
{{#external-link-to href=item.gitHubHref class='commit-branch-url'}}
  {{svg-jar item.icon class='icon'}}
  <span data-tooltip='View {{item.type}} on GitHub' class=…>{{upcase (:laughing:) item.type}} {{item.displayName}}</span>
  {{svg-jar 'icon-external-link' class='icon'}}
{{/external-link-to}}
```

This is obviously not a realistic implementation, but you can see how it could reduce complexity in the templates. I start to think about this when I see the same conditional scattered around a codebase. But I’m just pointing this out as something you may not have heard about, it seems still under control here, just getting toward refactoring-itch territory for me 😀

Typing that made me wonder whether line 54 should have a `commit-tag-url` branch? But maybe not, if they’re sharing styles.

Reply:
I also have the feeling that this could be done more elegantly. Eg having the request and tag icon conditional in and `event-icon` component. Also the polymorphism approach looks nice. 
So far this is all that tags are about, maybe it's fine?
I'll also leave `.commit-branch-url`, same styles and the test uses it, too.